pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -839,7 +839,12 @@ func (k *Bootstrapper) DeleteCluster(k8s config.KubernetesConfig) error {
 
 // SetupCerts sets up certificates within the cluster.
 func (k *Bootstrapper) SetupCerts(k8s config.KubernetesConfig, n config.Node) error {
-	_, err := bootstrapper.SetupCerts(k.c, k8s, n)
+	fs, err := bootstrapper.SetupCerts(k.c, k8s, n)
```

Comment:
If `bootstrapper.SetupCerts(k.c, k8s, n)` returns an error, `fs` slice is `nil`, and assets are not closed here. Neither they do inside `SetupCerts`